cmd/syncthing/cli/show.go

```diff
@@ -35,6 +35,11 @@ var showCommand = cli.Command{
 			Usage:  "Report about connections to other devices",
 			Action: expects(0, indexDumpOutput("system/connections")),
 		},
+		{
+			Name:   "discovery",
+			Usage:  "Report about the local discovery cache",
```

Comment:
Thanks for your contribution!

I know the docs contain the same sentence, however with (supposedly) more user interactions with this command it would be nice if it was a bit more descriptive. E.g. "Discovered addresses of remote devices (from cache of the running syncthing instance)". The cache bit is relevant, such that no-one thinks it looks it up actively e.g. from global discovery servers due to this method - "of the running syncthing instance" may be overly verbouse and not add anything, not sure - anyway all just a suggestions.